convene-web/app/models/invitation.rb

```diff
@@ -0,0 +1,28 @@
+class Invitation < ApplicationRecord
+  belongs_to :space
+
+  belongs_to :invitor, class_name: :Person
```

Comment:
We may also want to add a `has_many :invitations` on the Person class to indicate who they have invited